pkg/trace/schema/consensus.go

```diff
@@ -90,16 +89,42 @@ const (
 	// following schema:
 	//
 	//  | time  | height | timestamp |
-	BlockTable = "consensus_block_time"
+	BlockTable = "consensus_block"
+
+	// UnixMillisecondTimestampFieldKey is the name of the field that stores the timestamp in
+	// the last commit in unix milliseconds.
+	UnixMillisecondTimestampFieldKey = "unix_millisecond_timestamp"
```

Comment:
the default python libraries literally cannot parse the default golang timestamp string reliably!! apparently its too precise in that is occasionally will have a decimal for seconds with 9 digits, which overflows. had to switch to millisecond unix, but we could probably do nano